utils/serialize.go

```diff
@@ -9,7 +9,7 @@ func Serialize(data any) []byte {
 	switch castedData := data.(type) {
 	case uint64:
 		res := make([]byte, 8)
-		binary.LittleEndian.PutUint64(res, castedData)
+		binary.BigEndian.PutUint64(res, castedData)
```

Comment:
this can be a problem, we cant do this. if we have stuff saved on chain they will be desterilized wrongly

Reply:
done. see https://github.com/lavanet/lava/commit/ad7d7f8cc981dc891e09c8ddef560e7ea24b1d37